src/lua/api-gateway/validation/validatorsHandler.lua

```diff
@@ -185,7 +185,6 @@ function ValidatorsHandler:validateSubrequests(order, subrequests, localContext,
 
     if (subrequests_count == 0) then
         ngx.log(ngx.WARN, "Nothing to validate on subrequest")
-        ngx.exit(ngx.HTTP_OK)
```

Comment:
Can we add a unit/integration test that would demonstrate this indeed fixes the issue?